Java/src/main/java/epam/Airport.java

```diff
@@ -0,0 +1,90 @@
+package epam;
+
+import epam.Planes.experimentalPlane;
+import epam.models.MilitaryType;
+import epam.Planes.*;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+public class Airport {
+	private List<? extends Plane> planes;
+
+	public List<PassengerPlane> getPassengerPlane() {
+		return planes.stream().filter(plane -> plane instanceof PassengerPlane).map(
+				plane -> (PassengerPlane) plane).collect(Collectors.toList());
+	}
+
+	public List<MilitaryPlane> getMilitaryPlanes() {
+		return planes.stream().filter(plane -> plane instanceof MilitaryPlane).map(
+				plane -> (MilitaryPlane) plane).collect(Collectors.toList());
+	}
+
+	public PassengerPlane getPassengerPlaneWithMaxPassengersCapacity() {
+		return getPassengerPlane().stream().max(Comparator.comparing(PassengerPlane::getPassengersCapacity)).orElseThrow(
+				NoSuchElementException::new);
+	}
+
+	public List<MilitaryPlane> getTransportMilitaryPlanes() {
```

Comment:
you can use streams to get the result in this method, or you can split the method because you are getting the object list and filtering in the same method.

Reply:
Done